content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -23,11 +23,24 @@ sees.  The simplest way to do this is to run an interactive busybox Pod:
 ```none
 kubectl run -it --rm --restart=Never busybox --image=gcr.io/google-containers/busybox sh
 ```
-
 {{< note >}}
 If you don't see a command prompt, try pressing enter.
 {{< /note >}}
 
+Alternatively you can run the command directly from the pod. For example, let us say we wish to run nslookup for an existing service `servicesample` and store the output in nslookupout.txt. Then the command will be 
+
+```none
+kubectl run test-nslookup --image=busybox:1.28 --rm -it --restart=Never  -- nslookup servicesample > nslookupout.txt
+```
+
+Suppose you want to perform nslookup against an existing pod itself (say the pod ip is 10.244.1.13 in the default namespace), then the command would be 
```

Comment:
```suggestion
Suppose you want to perform nslookup against an existing pod itself. If the example pod's IPv4 address is 10.244.1.13, and the pod is in the `default` namespace, then you run:
```

Reply:
@sftim Thanks for the review Tim. the nslookup works only from versions starting 1.28 only. That is the reason, I used that version. 

Thanks for the corrections and feedback.